### ISSUE TITLE
Docstrings with content type are not properly formatted

### ISSUE THREAD

**Issue by bursauxa** [alexkrechik/VSCucumberAutoComplete#397]
Basic docstrings that contain json are very well handled by the extension, which detects that the contents are json and indents it as such. Below is an example after applying autoformat:

```cucumber
"""
{
  "property": "value"
}
"""
```

However, docstrings that are annotated with a content type are treated as raw strings, and smart formatting is lost. Below is the same example after applying autoformat, with content type provided:

```cucumber
"""json
{
"property": "value"
}
"""
```

In some cases, syntactic coloration is also broken beyond merely the docstring.

Note that content type on docstring is not described in the cucumber.io documentation, but has been confirmed as being part of the specification - see https://github.com/cucumber/docs/issues/640 where the issue was raised.

**Comment by Abhi347** [comment]
Just to add to the feature request that additional support for `yaml` format or `no-format` (probably `pre`)  for the docstring would be amazing too.

**Comment by alexkrechik** [comment]
It should be fixed after 2.16.0 release
